diwako_cbrn.vr/scripts/cbrn/functions/fn_postInit.sqf

```diff
@@ -1,248 +1,273 @@
 if (isServer) then {
-    publicVariable "cbrn_zoneSimulationRange";
-    cbrn_vehicles = cbrn_vehicles apply {
-        _x params [["_obj", "nothing", [""]], "_value"];
-        _obj = missionNamespace getVariable [_obj, _obj];
-        if !(_obj isEqualType "") then {
-            _obj = typeOf _obj;
-        };
-        [_obj, _value]
-    };
-    cbrn_vebiclesHash = createHashMapFromArray cbrn_vehicles;
-    {
-        _x params ["_class"];
-        [_class, "initPost",{
-            params ["_vehicle"];
-            _vehicle setVariable ["cbrn_proofing", cbrn_vebiclesHash getOrDefault [typeOf _vehicle, 0], true];
-        }, false, [], true]call CBA_fnc_addClassEventHandler;
-    } forEach cbrn_vehicles;
+	publicVariable "cbrn_zoneSimulationRange";
```

Comment:
Oh, that is a  lot of spaces to tabs converted. needs to be  again tabs -> spaces

Reply:
I'm going to mass convert tabs -> space for this file. There seems to be _something_ strange going on with this repo in particular, because some files just have spaces instead of tabs, even where it makes no logical sense...

Reply:
editorconfig should apply to all files except the mission sqm file. Anyways, I will go to bed, still massive thanks for doing this. always excited when people are willing to make pull requests.